src/components/Modal/index.tsx

```diff
@@ -29,6 +29,9 @@ const BaseModal = ({ title, modalShow, onHide, ModalBody }: BaseModalProps) => {
         <></>
       )}
       <Modal.Footer>
+        <Button id='modal-close-button' data-testid='modal-close-button'>
```

Comment:
Id must be unique

Reply:
LMAO i changed both without even seeing this comment, i think i know how you think now a bit more but yea updated !

Reply:
also i think it was failing cuz button didnt have onclick thing